### PR TITLE
Look mama, no hands!

### PR DESCRIPTION
**Pull request by traversaro**
@francesco-romano the relevant commit is : https://github.com/robotology/codyco-modules/commit/869cd8fcb47a91f002f4b2ba8f5b4ff2e2fba7bb 

I guess everything should be ok, but just in case.

### CODE REVIEW

File: src/modules/torqueBalancing/include/torquebalancing/TorqueBalancingController.h [hunk 1]
```diff
@@ -136,10 +136,10 @@ namespace codyco {
             Eigen::VectorXd m_desiredCentroidalMomentum;  /*!< 6 */
             Eigen::VectorXd m_desiredHandsForces; /*!< 12 */
```
Comment (francesco-romano) [inline src/modules/torqueBalancing/include/torquebalancing/TorqueBalancingController.h]: I think also this line can be removed

File: src/modules/torqueBalancing/include/torquebalancing/TorqueBalancingController.h [hunk 2]
```diff
@@ -136,10 +136,10 @@ namespace codyco {
...
             Eigen::VectorXd m_desiredContactForces; /*!< 12 (Vectorisation of contact wrenches at feet: left_wrench and right_wrench) */
-            
+
             bool m_leftHandForcesActive;
```
Comment (francesco-romano) [inline src/modules/torqueBalancing/include/torquebalancing/TorqueBalancingController.h]: Also these two..

File: src/modules/torqueBalancing/src/TorqueBalancingController.cpp [hunk 3]
```diff
@@ -318,21 +318,21 @@ namespace codyco {
...
 //            m_svdDecompositionOfCentroidalForceMatrix.compute(m_centroidalForceMatrix).solve(m_desiredCentroidalMomentum - m_gravityForce);
-            
+
             math::pseudoInverse(m_centroidalForceMatrix, m_svdDecompositionOfCentroidalForceMatrix,
                                 m_pseudoInverseOfCentroidalForceMatrix, PseudoInverseTolerance);
-            
+
             m_desiredFeetForces = m_pseudoInverseOfCentroidalForceMatrix * (m_desiredCentroidalMomentum
                                                                             - m_gravityForce
                                                                             - m_desiredHandsForces.head(6) - m_desiredHandsForces.tail(6));
```
Comment (francesco-romano) [inline src/modules/torqueBalancing/src/TorqueBalancingController.cpp]: DesiredHands now should be removed from the equation, right?

**Comment by francesco-romano** [comment]
I added some comments...
Can you see if they make sense?

**Comment by traversaro** [comment]
Ok thanks... looking into the comments. My rationale is that we can then reintroduce this force task in a link independent way (that will result also in a much cleaner and manteinable code).

**Comment by francesco-romano** [comment]
I totally agree. But maybe it makes more sense to remove the old code and start from scratch (only for the "task" side, not the whole balancing)

**Comment by francesco-romano** [comment]
Anyway: if it works we can merge this PR. (it is just that we are adding and subtracting zero in the force vector)

**Pull request closed without merging**